### PR TITLE
Feature/enemy range

### PR DESCRIPTION
**Pull request by matej172**
- gitignore + idea direcotry

- enemy range attribute

### CODE REVIEW

**Comment by KubkoFrano** [comment]
terrible

File: .gitignore [hunk 1]
```diff
@@ -9,7 +9,7 @@
 /[Bb]uilds/
 /[Ll]ogs/
 /[Mm]emoryCaptures/
-/.idea/
+/.idea
 
 # Never ignore Asset meta data
 !/[Aa]ssets/**/*.meta
```
Comment (AdamHostin) [inline .gitignore:15]: really

File: .gitignore [hunk 2]
```diff
@@ -9,6 +9,7 @@
 /[Bb]uilds/
 /[Ll]ogs/
 /[Mm]emoryCaptures/
+/.idea/
 
 # Never ignore Asset meta data
```
Comment (AdamHostin) [inline .gitignore]: idea?

**Review by AdamHostin — CHANGES_REQUESTED** [review]

**Comment by AdamHostin** [comment]
LevelUp demo

**Pull request closed without merging**